diffoscope/Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM python:3.7-slim-buster
+FROM python:3.9-slim-buster
```

Comment:
I'm a little hesitant on this one given they don't appear to explicitly claim they support 3.9 yet: https://salsa.debian.org/reproducible-builds/diffoscope/-/blob/b87e3b2204a73c94277b4869deee0e8250d97578/setup.py#L79-81

(I'm sure it likely works, but I'd rather not be their guinea pig :smile:)